documentation/asciidoc/microcontrollers/c_sdk/quick_start.adoc

```diff
@@ -0,0 +1,156 @@
+== Quick-start your own project
+
+NOTE: The following instructions are terse, and Linux-based only. For detailed steps, instructions for other platforms, and just in general, we recommend you see the https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf[Getting started with Raspberry Pi Pico] and https://datasheets.raspberrypi.com/pico/raspberry-pi-pico-c-sdk.pdf[Raspberry Pi Pico C/{cpp} SDK] books.
+
+Install CMake (at least version 3.13), and GCC cross compiler
+
+----
+$ sudo apt install cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
+----
+
+Set up your project to point to use the Raspberry Pi Pico SDK
+
+Either by cloning the SDK locally (most common) :
```

Comment:
As there are multiple steps involved, IMHO it's not clear where this "Either..." part starts and stops? (IYSWIM)

Reply:
Direct copy and paste from Graham's README. 🤷‍♂️

Reply:
Fair enough.

Reply:
Ah, looking at https://github.com/raspberrypi/pico-sdk/blob/master/README.md I guess the either/or sections are made more obvious by the use of bullet-point indentation, which you're not using here?